test/translateDatetime.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('chai');
+const fn = require('../api/util/translateDatetime.js');
+
+const roundTimestampToDay = (input, timezone) => Math.round(fn(input, timezone)
```

Comment:
can use within instead of a rounding function
http://www.chaijs.com/api/bdd/#method_within

Reply:
❔ 

Why would I use a BDD method to initialize this variable?

Reply:
noooo no no, instead of `expect round(a).to eq round(b)`, use something like `expect a.to be_within(b - 1.second, b + 1.second)`
should be easier to maintain and read

Reply:
But isn't that less precise than what's currently being verified?

Reply:
above was an example, can make the range as precise as you want.